### PR TITLE
[16.0][MIG] edi_record_metadata_oca: Migration to 16.0

### PR DESCRIPTION
**Pull request by QuocDuong1306**
Depends on:
- #5

### CODE REVIEW

**Comment by nilshamerlinck** [comment]
(pre-commit fails because of `oca-fix-manifest-website` hook that is not aware of the excludes directives; this will be sorted out once https://github.com/OCA/edi-framework/pull/5 is merged)

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo_test_helper
+odoo-addon-edi_oca @ git+https://github.com/OCA/edi-framework.git@refs/pull/5/head#subdirectory=setup/edi_oca
```
Comment (simahawk) [inline test-requirements.txt]: you can drop this

**Review by simahawk — COMMENTED** [review]
LG overall

**Comment by simahawk** [comment]
@QuocDuong1306 ping

**Comment by QuocDuong1306** [comment]
Thank @simahawk, it's done here

**Review by simahawk — APPROVED** [review]

**Pull request merged**